public/app/features/dashboard-scene/settings/AnnotationsEditView.tsx

```diff
@@ -133,6 +133,7 @@ function AnnotationsSettingsView({ model }: SceneComponentProps<AnnotationsEditV
   const { navModel, pageNav } = useDashboardEditPageNav(dashboard, model.getUrlKey());
   const { editIndex } = model.useState();
   const panels = dashboardSceneGraph.getVizPanels(dashboard);
+  const isSingleTopNav = config.featureToggles.singleTopNav;
```

Comment:
Quick question for better understanding: Why doesn't it make sense to define this only once e.g. in the constructor? (same question as for `public/app/features/dashboard-scene/settings/DashboardLinksEditView.tsx`)

Reply:
i don't think the constructor is common to both components 🤔 

i guess it _could_ live outside both components in the general file scope. it would cause a slight problem if you set feature toggles in local storage - you'd have to fully refresh the page instead of e.g. switching route. idk, it's super minor so probably wouldn't be an issue. but i think keeping those calls scoped to the components is safest